src/frontends/onnx/frontend/src/op/reduce.cpp

```diff
@@ -141,7 +141,7 @@ std::shared_ptr<ov::Node> make_ov_reduction_op(const Node& node,
 namespace set_1 {
 ov::OutputVector reduce_log_sum(const ov::frontend::onnx::Node& node) {
     const ov::Output<ov::Node> sum_node =
-        make_ov_reduction_op<v1::ReduceSum>(node, node.get_ov_inputs().at(0), supported_types_v1);
+        make_ov_reduction_op<v1::ReduceSum>(node, node.get_ov_inputs().at(0), supported_types_v2);
```

Comment:
Please, don't change it (https://github.com/onnx/onnx/blob/main/docs/Changelog.md#ReduceLogSum-1).

New types are available since opset-13 (https://github.com/onnx/onnx/blob/main/docs/Changelog.md#ReduceLogSum-13). Please, introduce it in opset-13.

Reply:
We have already discussed it in the comments above, please take a look at https://github.com/openvinotoolkit/openvino/pull/23508#discussion_r1531005884.
The subgraph is the same, so in such case different types restrictions between onnx versions are artificial from the ov perspective, and lead to code redundancy.